test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -35,6 +35,17 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
     }
+    
+    @Test
+    public void elementsContainNull()  {
```

Comment:
The method name can be `isAnyNull`. You can refer to our coding standard for naming convention for test methods: `featureUnderTest_testScenario_expectedBehavior()`, where second and third parts can be omitted.